Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM tomcat:9.0.37-jdk8-openjdk-slim
+FROM tomcat:9.0.50-jdk11-openjdk-slim
```

Comment:
Available [tomcat tags](https://hub.docker.com/_/tomcat?tab=tags) could be found in public Docker Hub.
I selected version 9.0.50 because upstream is using the same [tomcat version](https://github.com/GESoftware-CF/uaa/blob/master/dependencies.gradle#L21) for hosting UAA locally.

Reply:
Thanks, that information will be useful in the commit message for whoever in the future looks at this if we aren't around.

Reply:
Updated commit message.